src/Components/Toast/Toast.jsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from "react";
+import "./Toast.css";
+export const Toast = ({ message, type, handleclosetoast }) => {
+  useEffect(() => {
+    const time_to_show_toast = setTimeout(() => {
+      handleclosetoast();
+    }, 1500);
+    return () => {
+      clearTimeout(time_to_show_toast);
+    };
+  });
+  return (
+    <div className={`alert ${type}`}>
+      <p className="alert-text">{message}</p>
+      <a onClick={handleclosetoast} className="alert-cross">
```

Comment:
As suggested above `a` tag is not the best option here. It causes a page refresh unnecessarily

Reply:
Done